### PR TITLE
Add image to toolbar list

### PR DESCRIPTION
**Pull request by karelhala**
When toolbar item does not have icon, we use img_url of this item.

Add $event to view change click.

### CODE REVIEW

**Comment by karelhala** [comment]
We should update bower, npm version and create tag so we can reference these changes as `0.0.2` instead of branch name.

File: dist/js/ui-components.js [hunk 1]
```diff
@@ -301,6 +341,7 @@
 	        this.$window = $window;
 	        this.$location = $location;
 	        this.$sce = $sce;
+	        console.log(this.toolbarItems);
```
Comment (himdel) [inline dist/js/ui-components.js]: Mind regenerating the js? Pretty sure you must have added this manually..

Comment (karelhala) [inline dist/js/ui-components.js]: Yeah removed the console.log but, forgot to rebuild the project before pushing. Good catch.

**Comment by himdel** [comment]
LGTM ,merging and releasing 0.0.2..

**Pull request merged**